README.md

```diff
@@ -1,7 +1,14 @@
-# Tauri + React + Typescript
+# ACE v3
+The third iteration of the Advanced Cockpit Emulator.
 
-This template should help get you started developing with Tauri, React and Typescript in Vite.
+## Installation and usage
+1. [Install Rust and toolchain](https://tauri.app/v1/guides/getting-started/prerequisites)
+2. Clone the repo and `cd` into it
+3. `npm i`
+4. `corepack enable` - Tauri wants `yarn`, `corepack` helps to manage all the different package managers. 
+    Comes with Node 16.10+. You can install `yarn` manually if don't want to use `corepack` too.
+5. `npm start`
```

Comment:
Tauri has no preference between `yarn` and `npm`. Both work perfectly fine, and I think we can forego the whole `corepack` thing for simplicity.

Reply:
Just noticed that yarn was used in tauri config. Fixed that, should be simpler now.